### PR TITLE
fix bug: PAFPN has no attribute extra_convs_on_inputs

### PR DESCRIPTION
**Pull request by luuuyi**
PAFPN has no attribute extra_convs_on_inputs

retinanet-like detectors can use PAFPN

### CODE REVIEW

File: mmdet/models/necks/pafpn.py [hunk 1]
```diff
@@ -124,11 +124,13 @@ def forward(self, inputs):
...
             # add conv layers on top of original feature maps (RetinaNet)
             else:
-                if self.extra_convs_on_inputs:
+                if self.add_extra_convs == 'on_input':
                     orig = inputs[self.backbone_end_level - 1]
                     outs.append(self.fpn_convs[used_backbone_levels](orig))
-                else:
+                elif self.add_extra_convs == 'on_output':
                     outs.append(self.fpn_convs[used_backbone_levels](outs[-1]))
+                else:
```
Comment (hellock) [inline mmdet/models/necks/pafpn.py:132]: `on_lateral` can also be implemeted

**Comment by hellock** [comment]
Could you fix the linting error?

**Comment by luuuyi** [comment]
> Could you fix the linting error?

sorry, the first lint error is some code line is too long. Ifix it, but the second lint error is  `yapf: files were modified by this hook`. I have no idea how to fix it. Could you help me?

**Comment by hellock** [comment]
It can be merged since the linting error has been fixed. You can just ignore the errors appearing in `build_parrots`.

**Review by hellock — APPROVED** [review]

**Pull request merged**